_layouts/index.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: base
+---
+
+
+<!-- style="background-color: blue;" -->
+  <div class="container content {{ layout.css }} {{ page.css }}">
+        <!-- section1 START-->
+        <div class="columns section1 has-text-centered-mobile is-vcentered is-flex" >
+            <div class="column section1_text">
+                <p class="hallo_text">Hallo! Everybody!</p>
+                <h1 class="is-size-3-mobile welcome_text">Welcome to our online & interactive game
+                    on DNA data analysis</h1>
+                <p class="come_join_text">Learn Bioinformatics online through interesting games! Come join us now!</p>
+                <form action="{{ site.baseurl }}/game/intro/">   
+                    <button type="submit">
+                        Get Started
+                    </button>
+                </form>
+            </div>
+            <div class="column section1_img is-hidden-mobile">
+                <img src="{{ site.baseurl }}/images/hero_img.png" alt="">
+            </div>
+        </div>
+        <!-- section1 END-->
+
+        <!-- section2 START-->
+        <div class="section2 has-text-centered-mobile">
+            <div class="section2_text">
```

Comment:
What is `section2` class?  It cut the sentence in my notebook screen and leave a big empty space

![image](https://user-images.githubusercontent.com/1842467/168807529-ba7e7e30-6079-4b5c-a0d9-42a289b14912.png)